### DISCUSSION TITLE
Gameplay pause screen doesn't support legacy skins

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#17554] [Q&A]
It seems that this is one of the last elements of legacy skins that have not yet been implemented/supported into the lazer. 

There are only two of these elements, it seems to me. I already wrote about "section_pass/fail" a couple of years ago (https://github.com/ppy/osu/issues/9783)

**Comment by peppy** [comment]
Are you talking about the pause/retry button sounds? If so, it's pretty obvious why they aren't implemented (the pause screen itself doesn't support skinning).

**Reply by Moltanicaa** [reply]
Yes, about it. Does this also apply to "section pass/fail"?

**Reply by peppy** [reply]
That is a very different element. I'm not sure what you mean.

**Reply by Moltanicaa** [reply]
I know, I just noticed that of all the elements of the skins, only the pause screen remained (with all the sounds and custom image support), as well as the "section_pass" and "section_fail" elements (also with sounds, they are not in the current version).  That's why I mentioned them together, but I'm more worried about "section_pass/fail" since it's (seemingly) a game element, but the legacy version is completely omitted and replaced by a new one.
